### ISSUE TITLE
Slow dropdown list opening on empty

### ISSUE THREAD

**Issue by ArekSliwa** [angular-material-extensions/select-country#17]
**Describe the bug**
When the component is empty and the user clicks to open, the dropdown list is opening after a few seconds (2 or 3) which feels buggy.

**To Reproduce**
1. Go to demo page https://angular-material-extensions.github.io/select-country/
2. Click on the select component to open list
3. And you will see short time that seems like select is not reacting on click

**Expected behavior**
I would like to open it immediately. 

**Desktop (please complete the following information):**
 - OS: OSX
 - Browser: Chrome
 - Version:  the newest

**Additional context**
If this is difficult to fix, can we at least open immediately dropdown with loader? It will be more user friendly

**Comment by Yug0** [comment]
@ArekSliwa I have made a pull request to resolve that issue.
Waiting for @AnthonyNahas  to review it.

**Comment by ArekSliwa** [comment]
> @ArekSliwa I have made a pull request to resolve that issue.
> Waiting for @AnthonyNahas to review it.

big thanks

**Comment by Yug0** [comment]
> > @ArekSliwa I have made a pull request to resolve that issue.
> > Waiting for @AnthonyNahas to review it.
> 
> big thanks. So should I close issue already?

I would wait a bit, to check if it will be good enough solution.

**Comment by AnthonyNahas** [comment]
Did someone test #16 ? 

I can't confirm that the PR helped to fix the issue...

@Yug0 what do you think ?

**Comment by ArekSliwa** [comment]
I tested and can't confirm also

**Comment by Yug0** [comment]
> I tested and can't confirm also

Hi @ArekSliwa, did you add parameter **itemsLoadSize** like i mentioned?

**Comment by ArekSliwa** [comment]
ah sorry, no I will check.

**Comment by itsoisi** [comment]
is there an update how to fix this ?

**Comment by KingDarBoja** [comment]
@itsoisi yes, the linked pull request could fix it if there are enough testers that can confirm it.

**Comment by AnthonyNahas** [comment]
@KingDarBoja Thanks for the PR #16 

I've just published [v2.2.0](https://www.npmjs.com/package/@angular-material-extensions/select-country) that contains this PR and that improves the performance.

Please use `itemsLoadSize` property to choose the number of countries that should be fetched


e.g:

```html
<mat-select-country appearance="outline"
                        label="Country"
                        [itemsLoadSize]="5"
                        (onCountrySelected)="onCountrySelected($event)">
    </mat-select-country>
`
``



cheers 🍻
